rules-tests/Php71/Rector/FuncCall/CountOnNullRector/Fixture/count_on_class_method_stmts.php.inc

```diff
@@ -1,31 +0,0 @@
-<?php
-
-namespace Rector\Tests\Php71\Rector\FuncCall\CountOnNullRector\Fixture;
-
-use PhpParser\Node\Stmt\ClassMethod;
-
-final class CountOnClassMethodStmts
-{
-    public function run(ClassMethod $classMethod)
-    {
-        return count($classMethod->stmts) === 5;
-    }
-}
-
-?>
------
-<?php
-
-namespace Rector\Tests\Php71\Rector\FuncCall\CountOnNullRector\Fixture;
-
-use PhpParser\Node\Stmt\ClassMethod;
-
-final class CountOnClassMethodStmts
-{
-    public function run(ClassMethod $classMethod)
-    {
-        return count((array) $classMethod->stmts) === 5;
```

Comment:
`$classMethod->stmts` can be null, so `(array)` cast is neeed.

Reply:
I'll fix this one